### ISSUE TITLE
Remove menu item shortcut

### ISSUE THREAD

**Issue by Perelandra0x309** [humdingerb/clipdinger#12]
Should the Remove menu items in the right click menus for the History and Favorites entries show the DEL key shortcut since that is a viable key stroke?

**Comment by humdingerb** [comment]
Would be nice. Unfortunately, AFAIK every shortcut used in menu items will have an "ALT" command key added. If you know a simple, not too hackish way to add shortcut without command key, I'm a all ears.

**Comment by Perelandra0x309** [comment]
Yes I guess you're right since the DEL isn't actually a shortcut, just an alternate input to achieve the same result.  What if you derive your own MenuItem class and override the DrawContent() function to put a DEL icon on the right?  Create a bitmap of the icon and draw it like:
https://github.com/Perelandra0x309/einsteinium/blob/master/src/Launcher/AppMenuItem.cpp#L83

**Comment by humdingerb** [comment]
I'm not prepared to jump through those hoops. I think I'll wait for Haiku'S API to catch up and allow non-CMD key shortcuts for menu items. )
